quickwit/quickwit-config/src/node_config/mod.rs

```diff
@@ -226,6 +226,8 @@ pub struct SearcherConfig {
     #[serde(default)]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub storage_timeout_policy: Option<StorageTimeoutPolicy>,
+    pub warmup_memory_budget: ByteSize,
```

Comment:
Can we define a default value for this?

And reuse the default value function in SearcherConfig::default for consistency

Reply:
Should I also do the same for all the other fields of the same struct? https://github.com/quickwit-oss/quickwit/blob/c3de281064deb42dc19554291500957b314baf0c/quickwit/quickwit-config/src/node_config/mod.rs#L211-L218

Reply:
Yes, but does it apply to the individual attributes? It really works that way?

Reply:
just tested. yep that works that way.